### ISSUE TITLE
60 fps 4.5 GB file cannot be recovered

### ISSUE THREAD

**Issue by Keksmania** [Keksmania/FileToYoutube#8]
60 fps 4.5 GB file cannot be recovered. About 20 volume files out of 1655 are incorrect.

I am testing if it can be solved by saving images as yuv420 instead of pal8.. even though the images only contain black and white the youtube compression will create shades of gray. I tried pal8 as a compromise between file size and accuracy.

**Comment by Keksmania** [comment]
There is no difference between RGB24 and Pal8. I will keep pal8 for the reduced file size. The video from YouTube is the problem. Im still not sure if the problem only occurs on 60fps video. I need to test the 4.5GB file in 6 fps for example

**Comment by Keksmania** [comment]
im currently testing around with different qr code ecc levels. 
Medium with CRF 30 does not seem to work either in some cases (especially in big files)
Right now im trying Medium CRF 27 and CRF 15.
Also im testing ECC level low with CRF 27..

**Comment by Keksmania** [comment]
All of the tested settings worked... 
I have a feeling that the problem lies in the way batch images are created. I changed the setting to always create a full 10x10 batch image without any black frames. Since having black frames reduces the local bitrate and reduces the overall average bitrate it is possible that YouTube will compress more than it would without the black frames. 

I will try a the 4.5 GB file mentioned above with the new settings:
`CRF 27, Low ECC, No completely black frames. 
`

**Comment by Keksmania** [comment]
It is almost impossible to make sure that 160k files can be recovered. Therefore I needed to "think different" about the problem.  What if we instead want to recover most files but not all and then repair the rest with some kind of parity? Yes we can do exactly that by creating a Raid-Like System using parchive. In the next release this issue will be solved. I already conducted early tests and it seems to work! All I need to do is do the same with the 4.5 file mentioned above to verify my claims. 
Because testing was very slow since generating thousands of qr codes was too slow I changed the library to create QR-Codes. Now it is way faster than before. We also now work only with Byte Arrays instead of working Strings that represent Byte Arrays
